.github/workflows/release-process.yml

```diff
@@ -40,10 +40,21 @@ jobs:
           cd cli
           ./hack/build-binaries.sh
           shasum -a 256 ./kctrl-* | tee -a ../tmp/checksums.txt
-          echo "# :open_file_folder: Files Checksum" | tee ../tmp/checksums-formatted.txt
-          echo '```' | tee -a ../tmp/checksums-formatted.txt
-          cat ../tmp/checksums.txt | tee -a ../tmp/checksums-formatted.txt
-          echo '```' | tee -a ../tmp/checksums-formatted.txt
+
+      - name: Run Package build
+        run: |
+          constraintVersion="${{ github.ref_name }}"
+          ./cli/kctrl-linux-amd64 pkg release -y -v ${constraintVersion:1} --debug
```

Comment:
I'm assuming this line means we want to run the package release command with the binary we just generated as part of this release? (as opposed to the previously released version). I think that makes sense to me

Reply:
Yes, that's correct.